_pytest/tmpdir.py

```diff
@@ -54,9 +54,8 @@ def getbasetemp(self):
             else:
                 # use a sub-directory in the temproot to speed-up
                 # make_numbered_dir() call
-                import getpass
                 temproot = py.path.local.get_temproot()
-                rootdir = temproot.join('pytest-of-%s' % getpass.getuser())
+                rootdir = temproot.join('pytest-of-%s' % get_user())
```

Comment:
That actually looks odd to me in combination with using a fixed ``pytest-tox``.  If you have a Jenkins machine that executes multiple tox runs is this not bound to lead to bad interactions? 

What about simply not creating a subdir but using the "slow" make_numbered_dir on the temproot?